### PR TITLE
Proposal for not requiring the `get_learning_rate` function + test

### PR DESCRIPTION
**Pull request by rhaps0dy**
Here's how I think learning rate should be implemented so we can log it.

We must use `inject_hyperparameter` with the *top level* optimizer object, so that we can get it directly from the dict. The result has fewer lines of code.

### CODE REVIEW

File: cleanba/cleanba_impala.py [hunk 1]
```diff
@@ -871,7 +837,8 @@ def train(
 
                 writer.add_scalar("policy_versions/learner", args.learner_policy_version, global_step)
 
-                lr = get_learning_rate(unreplicate(agent_state.opt_state))
+                print(agent_state.opt_state.hyperparams)
```
Comment (taufeeque9) [inline cleanba/cleanba_impala.py:840]:
```suggestion
```
Remove printing all hyperparams?

**Review by taufeeque9 — APPROVED** [review]
Thanks for simplifying the learning rate logging process! I had a feeling when I was writing the `get_learning_rate` function that there must be a better way to fetch it.
